test/support/docker/Dockerfile

```diff
@@ -3,7 +3,8 @@ FROM alpine:3.5
 # Set up an Alpine Linux machine running an SSH server.
 # Autogenerate missing host keys.
 
-RUN apk add --no-cache openssh sudo
+RUN echo http://dl-cdn.alpinelinux.org/alpine/edge/main >> /etc/apk/repositories
+RUN apk add --update --no-cache openssh sudo perl-utils
```

Comment:
– see also: https://en.wikipedia.org/wiki/Sha1sum (didn't know that `sha1sum` actually seems to be the more common case, rather than `shasum`, on Linux)